joint_limits/src/joint_limiter_interface.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// \author Dr. Denis Stogl
+
+#include "joint_limits/joint_limiter_interface.hpp"
+
+namespace joint_limits
+{
+}  // namespace joint_limits
```

Comment:
I will test this one more time and explain way.

Reply:
OK, we are required to have this `.cpp`file as this is a base interface for `pluginlib`. I have added the full implementation to `.hpp` because of templated class, as the exact methods are generated on the compile time.